From b80108f01b1425427f98341168ea44c4b1a5a2cf Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 28 Jun 2019 16:02:13 +0800
Subject: [PATCH] use lxdm to replace dm-tool

OE does not support lightdm but lxdm,
so use lxdm to replace dm-tool

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for xfce4-panel 4.18.3.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for xfce4-panel 4.20.0.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

---
 plugins/actions/actions.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/actions/actions.c b/plugins/actions/actions.c
index 5e58078..b16d38e 100644
--- a/plugins/actions/actions.c
+++ b/plugins/actions/actions.c
@@ -1054,11 +1054,11 @@ actions_plugin_actions_allowed (ActionsPlugin *plugin)
       g_free (path);
     }
 
-  path = g_find_program_in_path ("dm-tool");
+  path = g_find_program_in_path ("lxdm");
   if (path != NULL)
     {
       PANEL_SET_FLAG (allow_mask, ACTION_TYPE_SWITCH_USER);
-      plugin->switch_user_cmd = "dm-tool switch-to-greeter";
+      plugin->switch_user_cmd = "lxdm -c USER_SWITCH";
     }
   else
     {
-- 
2.8.1
